apis/v1alpha1/gateway_types.go

```diff
@@ -281,6 +283,20 @@ type Listener struct {
 	// are compatible with the application protocol specified in
 	// the Protocol field.
 	//
+	// Multiple Routes may match a request to this Listener. If that happens,
```

Comment:
I'm not quite sure how to parse "Multiple Routes may match a request to this Listener". Does it mean that a given client request could be matched to different routes?

Reply:
This was confusing, I've tried rewording it but still not quite happy with the end result, open to options. I'm trying to communicate that multiple rules may technically match a request but only one may be selected.